stdlib/public/TensorFlow/CompilerRuntime.swift

```diff
@@ -116,6 +128,11 @@ private class TraceContext {
   /// The trace graph function created by `finalize()`.
   var traceGraphFn: CTFFunction?
 
+  /// The number of additional input tensors to the trace graph function,
+  /// created from concrete intermediate tensors in the tracee, such as `y` in
+  /// the code snippet above.
+  var additionalInputTensorCount: Int32 = -1
```

Comment:
This increases the chance of catching bugs (since `-1` is never valid), if we forget to set it to some legal value.

Reply:
I see where you are coming from, however, there's a safer way to do that in Swift -- since you expect `additionalInputTensorCount` to always be set during class initialization, removing the default value makes you able to catch initialization bugs at compile-time. When you forget to set it in `init`, Swift gives you a compile-time error saying "member is not initialized", which is significantly better than catching this at runtime.